lms/djangoapps/verify_student/urls.py

```diff
@@ -152,3 +152,11 @@
         name="verify_student_incourse_reverify"
     ),
 )
+
+if settings.FEATURES.get('ENABLE_SOFTWARE_SECURE_FAKE'):
+    # Fake response page for incourse reverification ( software secure )
+    from verify_student.tests.fake_software_secure import SoftWareSecureFakeView
```

Comment:
Ah, I see -- didn't realize this was in urls.py.  I'm okay with the inline import here.